### ISSUE TITLE
Websocket reconnecting after failure 

### ISSUE THREAD

**Issue by heekah7** [mickomagallanes/kucoin-futures-node-api#9]
Hi, I think we needed web socket reconnection upon failure badly, as order book will fails sometimes.

**Comment by tuloski** [comment]
Yes I agree. I'm opening 4 ws and statistically in around half an hour at least one fails and it doesn't reconnect. So annoying.

**Comment by tuloski** [comment]
I forked and modified to:
-handle disconnection and automatic reconnect
-ws is separate class now which can be instantiated multiple times
-each ws now supports multiple topic subscription
-working but probably not 100% right now

https://github.com/tuloski/kucoin-futures-node-api
